sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_clients_daily_v1/query.sql

```diff
@@ -28,64 +32,99 @@ WITH combined_urlbar_picked AS (
     [
       STRUCT(
         "autofill" AS type,
-        one_index_array(scalar_parent_urlbar_picked_autofill_sum) AS position
+        one_index_array(
+          replace_outlier_values_with_zero(scalar_parent_urlbar_picked_autofill_sum)
```

Comment:
How about calling `replace_outlier_values_with_zero` in the `one_index_array` function?
This would make these calls in the query shorter. `one_index_array` would need to be renamed to make it clearer what it does (e.g. `one_index_array_normalized`).